frontend/schema/class-schema-howto.php

```diff
@@ -240,4 +240,18 @@ protected function get_image_schema( $url ) {
 
 		return $image->generate_from_url( $url );
 	}
+
+	/**
+	 * Gets the post title.
+	 *
+	 * @return string $post_title The post title.
+	 */
+	protected function get_post_title() {
+		$post_title = wp_strip_all_tags( trim( get_the_title() ) );
```

Comment:
We have the Post's ID in our context object, so I'd suggest adding it in our call to here:

```suggestion
		$post_title = wp_strip_all_tags( trim( get_the_title( $this->context->id ) ) );
```